src/Platform/Platform.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
       <PackageReference Include="Particular.CodeRules" Version="0.8.0" PrivateAssets="All" />
-      <PackageReference Include="Particular.PlatformSample" Version="2.*" />
+      <PackageReference Include="Particular.PlatformSample" Version="2.34.0" />
   </ItemGroup>

```

Comment:
What is the need to force the sample to 2.34?

Reply:
For some reason, on my machine, I get a learning transport initialization error and I was trying to figure out if this is caused by the newest version of the PlatformSample package.